extract_unique_sensors_01.py

```diff
@@ -13,45 +17,59 @@ def create_parser():
 
     parser = argparse.ArgumentParser(description='Extract unique sensors')
     parser.add_argument('--file_name', help='File to extract unique sensors from.')
-    parser.add_argument('--url', help='A reference to SensorList.txt that specifies its ' 
-                        'location on a computer network.') 
+    parser.add_argument(
+        '--url',
+        help='A reference to SensorList.txt that specifies its location on a computer'
+        'network.'
+    )
     parser.add_argument('--kat_sensor', required=True, help='Name of unique sensor')
+    parser.add_argument('-v', metavar='verbosity', type=int, default=2, help='Logging'
+                        'verbosity: 0 -critical, 1- error, 2 -warning, 3 -info, 4 -debug')
+
     args = parser.parse_args()
     return args
 
+
 def read_sensors(file_name, url):
     """This function opens, reads and extracts the contents of the file_name.If the file_name
-    does not exist,with try will attempt to read the file_name from the url expecting it to be
-    there. If not, catch exception and perform a fallback.
+    does not exist,with try will attempt to read the file_name from the url expecting it
+    to be there. If not, catch exception and perform a fallback.
 
     Params
     ------
-    file_name: str        
+    file_name: str
         The actual name of the file.
-    url: str                       
+    url: str
         The location of SensorList.txt on the internet.
-                                                    
+
     Returns
     -------
     sensors_data: list
-        list of sensors. 
-    """                 
+        list of sensors.
+    """
     assert isinstance(file_name, str), "Expected a string."
-    assert isinstance(url, str), "Expected a string."                                          
+    assert isinstance(url, str), "Expected a string."
     sensors_data = []
     if file_name and Path(file_name).exists():
-        file_path = Path(file_name)           
+        file_path = Path(file_name)
+        logging.debug("Accesing sensor data from the file_name.")
         with open(file_path) as lines:
             sensors_data = lines.readlines()
-    elif url:                               
-        try: 
+    elif url:
+        try:
             sensors_url = urlopen(url)
             sensors_data = sensors_url.readlines()
-        except Exception:                         
-            print("Failed to retrieve sensor data")
-    return sensors_data  
- 
-def extract_all_sensors(sensors_data, sensor_name): 
+            logging.debug("Accessing sensor data from the url.")
```

Comment:
See comment: https://github.com/mamkhari/extract_sensor_names/pull/12/files#r478458777

Reply:
I don't follow. I thought I addressed this, logged when url is being used.

Reply:
Perhaps my comment was a bit vague...
I meant, please do the same as you did on the debug log that report `file_name`.

Reply:
Hint: `logging.debug("Accesing sensor data from the file_name: %r.", file_name)`

Reply:
Oh maybe I'm a bit slow to understand. Thank you.